spec/models/user_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe User do
+  it { should validate_presence_of(:first_name) }
+  it { should validate_presence_of(:last_name) }
+  it { should validate_inclusion_of(:role)
+         .in_array(%w{admin stockholder manager}) }
```

Comment:
Expression at 7, 51 should be on its own line.<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 7) spaces for indenting an expression spanning multiple lines.